src/main/java/ch/liquidmind/inflection/proxy/ProxyGenerator.java

```diff
@@ -40,6 +40,10 @@ public static void main( String[] args )
 		for ( String taxonomyName : taxonomyNames )
 		{
 			Taxonomy taxonomy = TaxonomyLoader.getContextTaxonomyLoader().loadTaxonomy( taxonomyName );
+			if (taxonomy == null) {
```

Comment:
I'm not sure that this is the way I would handle it; I may introduce a general purpose ProxyGeneratorException (and InflectionCompilerException), but I need to go through the code and have a good think. I appreciate your initiative, but could you take it out for now? You're welcome to define a test that fails so that don't forget about it.